packages/core/src/integrations/extraerrordata.ts

```diff
@@ -97,7 +103,7 @@ function _extractErrorData(error: ExtendedError, captureErrorCause: boolean): Re
         continue;
       }
       const value = error[key];
-      extraErrorInfo[key] = isError(value) ? value.toString() : value;
+      extraErrorInfo[key] = truncate(isError(value) ? value.toString() : value, maxValueLength);
```

Comment:
Actually now that I'm looking at this again, `const value = error[key]` means that `value` could be anything. What is the cleanest way here to make sure we truncate all the string? Should I adjust the `normalize` function?

Reply:
I think we can just do something like this:

```suggestion
      extraErrorInfo[key] = isError(value) || typeof value === 'string' ? truncate(`${value}`, maxValueLength) : value;
```

?

Reply:
The issue here is that it'll only truncate string values at the top-most level. String values that are nested within an object will not get truncated.

Reply:
Hmm I see. Yeah then if we want to fix this I guess we have to update normalize to also handle max value length!

Reply:
Given the normalize update will mean refactoring how base client and event processing works, I'm electing for the quick fix and just using your change on the `extraErrorInfo[key]` setter. This also unblocks the user.

I will create a GH issue and write down what we need to do to make a more robust fix.